vita3k/config/src/config.cpp

```diff
@@ -249,6 +266,8 @@ void merge_configs(Config &lhs, const Config &rhs, const std::string &new_pref_p
         lhs.hardware_flip = rhs.hardware_flip;
     if (lhs.performance_overlay != rhs.performance_overlay && (!init || rhs.performance_overlay))
         lhs.performance_overlay = rhs.performance_overlay;
+    if (lhs.backend != rhs.backend && (!init || rhs.backend != -1))
```

Comment:
```suggestion
    if (lhs.backend != rhs.backend && (!init || rhs.backend != 0))
```
i no understund this -1 is totaly wrong, is "0"